File: Java/프로그래머스/0/181874. A 강조하기/A 강조하기.java
class Solution {
    public String solution(String myString) {
        char ch[]=myString.toCharArray();
        for(int i=0; i<ch.length ;i++){
            if(ch[i]=='a') ch[i]=Character.toUpperCase(ch[i]);
            else if(ch[i]>'A' && ch[i]<'Z') ch[i]=Character.toLowerCase(ch[i]);
        }
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181874. A 강조하기/A 강조하기.js
function solution(myString) {
    var answer = '';
    var arr ='';
    for(var i=0; i<myString.length; i++){
        arr+=myString.charAt(i).toLowerCase();
    }
    for(var i=0; i<arr.length ; i++){
        if(arr.charAt(i)=='a'){
            answer+=arr.charAt(i).toUpperCase();
        }
        else {
            answer+=arr.charAt(i)
        }
    }
    return answer;
}